source/data/landings.yaml

```diff
@@ -413,8 +415,6 @@
               url: '/hosts-file/'
             - text: "HTTPS on Pantheon's Global CDN"
               url: '/https/'
-            - text: 'Configure Redirects'
-              url: '/redirects/'
```

Comment:
FYI to whoever copy-reviews this: Removed because this page was listed twice on the same landing, right next to itself.